settings.py

```diff
@@ -21,6 +21,6 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
```

Comment:
This needs for filling the database from "cinema_db_data.json", however, it has no effect on final tests.